### ISSUE TITLE
Release plan for the future ..

### ISSUE THREAD

**Issue by skx** [skx/rss2email#86]
Maybe I'm over-thinking this, but it would be nice to be explicit for the future.

* A while back we changed the configuration-file to support per-feed options.
  * When we run we'll upgrade in-place, if we need to, for compatibility.
* #85 will change the format of our seen/state markers.
  * In the past we had flat files, now we use a Key=Value database.
  * Once again we migrate in-place.

The current release is `release-2.5`, the next release `release-2.6` will have the BoltDB changes in it.  At some point we're going to drop the migration, and I guess `release-3.0` is the place to do it:

* If users upgrade from the latest available release to `release-3.0`, everything is fine.
* If users skip from `release-2.5`, or below, to release-3.0 their feeds will be 100% seen as unread/new.
* I guess also we'll drop the configuration upgrade too.

TLDR:

* Next release will be `2.6`.  It will have the BoltDB changes.
* There will probably be a 2.7, and 2.8 release in the future.
* All backwards-compatibility options will be dropped in `release-3.0`.
  * Users who upgrade from 2.6/2.7/2.8 will be fine.
  * Users who upgrade to 3.0 from an earlier release will see their emails duplicated.

**Comment by skx** [comment]
The current release is `release-2.7`.  The next release will be `release-3.0`:

* The legacy feed-paths will be ignored.
* The legacy configuration file will be ignored.

Both of these changes will simplify the code.

**Comment by skx** [comment]
Closed in #100.
